README.md

```diff
@@ -93,6 +93,49 @@ It is also possible for you to create a simple file upload button which yields t
 {{/file-upload}}
 ```
 
+The examples above using angle bracket invocation, available in Ember 3.4+
+
+```handlebars
+<FileDropzone @name={{"photos"}} as |dropzone queue|>
+  {{#if dropzone.active}}
+    {{#if dropzone.valid}}
+      Drop to upload
+    {{else}}
+      Invalid
+    {{/if}}
+  {{else if queue.files.length}}
+    Uploading {{queue.files.length}} files. ({{queue.progress}}%)
+  {{else}}
+    <h4>Upload Images</h4>
+    <p>
+      {{#if dropzone.supported}}
+        Drag and drop images onto this area to upload them or
+      {{/if}}
+      <FileUpload @name={{"photos"}}
+                  @for={{"upload-photo"}}
+                  @accept={{"image/*"}}
+                  @multiple={{true}}
+                  @onfileadd={{route-action "uploadImage"}} >
```

Comment:
`route-action` is not an encouraged pattern, so probably should not be used in examples.

Reply:
Probably should update the above example to remove it as well.

Reply:
route-action => action both examples